tenants/ai-example/lmeval-lab/base/serving-runtime.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  annotations:
+    opendatahub.io/accelerator-name: ""
+    opendatahub.io/apiProtocol: REST
+    opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+    opendatahub.io/template-display-name: Qwen2.5-0.5B-Instruct for TrustyAI LMEval lab
+    opendatahub.io/template-name: qwen-runtime
+    openshift.io/display-name: Qwen2.5-0.5B-Instruct for TrustyAI LMEval lab
+  labels:
+    opendatahub.io/dashboard: "true"
+  name: qwen-instruct
+spec:
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "8080"
+    openshift.io/display-name: Qwen2.5-0.5B-Instruct for TrustyAI LMEval lab
+  labels:
+    opendatahub.io/dashboard: "true"
+  containers:
+  - name: kserve-container
+    image: quay.io/opendatahub/vllm:stable-849f0f5
+    command:
+      - python
+      - "-m"
+      - vllm.entrypoints.openai.api_server
+    args:
+      - "--port=8080"
+      - "--model=/mnt/models"
+      - "--served-model-name={{.Name}}"
+      - "--dtype=float16"
+      - "--enforce-eager"
```

Comment:
Would recommend moving this to the inferenceService

Reply:
Not sure why it needs to be moved - all existing example models have args defined only in servingRuntimes.
But anyway - this is done.